pulsator4droid/src/main/java/pl/bclogic/pulsator4droid/library/circle/PulseCircle.java

```diff
@@ -0,0 +1,30 @@
+package pl.bclogic.pulsator4droid.library.circle;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+
+import pl.bclogic.pulsator4droid.library.PulseShape;
+
+public class PulseCircle implements PulseShape {
+
+    float mCenterX;
```

Comment:
these fields should be private, or have getters. or is this some kind of a best practice?

Reply:
There have been direct accesses somewhere around the library before, probably replaced by getters. Will do the same here.

Reply:
done